view-user/src/components/common/NumberCardLarge/NumberCardLarge.module.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 62.5vw;
+  height: 62.5vw;
+  border: 1.25vw solid #ff3342;
+  background: linear-gradient(135deg, #FFF 50%, #FFEEB1 51%);
+  border-radius: 3.125vw;
+  box-sizing: border-box;
+  box-shadow: 0.9375vw 0.9375vw 0px #ff3342;
+}
```

Comment:
Figmaのデザインから少し修正してみました。

![image](https://github.com/user-attachments/assets/8b70741f-cf91-418b-99c5-8976c940a46b)

```suggestion
.container {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 62.5vw;
  height: 62.5vw;
  border: 2vw solid #ff3342;
  background: linear-gradient(135deg, #FFF 50%, #FFEEB1 51%);
  border-radius: 7vw;
  box-sizing: border-box;
  box-shadow: 1vw 1vw 0vw 0.9vw #ff3342;
}
```